hicommonwealth/commonwealth
# Disconnect from a single community

## Link to Issue
Closes: #8710

## Description of Changes
- Implemented validation to prevent a user from disconnecting if they are the last admin of the community.
- Added a check for admin status in the deleteAddress function:
  - If the user is the last admin, a toast prompts them to invite and assign another admin before disconnecting.
  - If the user attempts to disconnect a magic wallet address, a toast informs them that it cannot be removed.
- Updated relevant error messages to provide clear feedback to the user.


## "How We Fixed It"

## Test Plan
- Go to Edit Profile Page.
- Navigate to Linked addresses section
- disconnect address where there is only one admin of that community.
- Verify that it prompts to invite some one before leaving. 

## Deployment Plan

1. 

## Other Considerations

-
